kafka/consumer.c

```diff
@@ -788,3 +788,30 @@ lua_consumer_dump_conf(struct lua_State *L) {
         return lua_librdkafka_dump_conf(L, (*consumer_p)->rd_consumer);
     return 0;
 }
+
+int
+lua_consumer_metadata(struct lua_State *L) {
+    consumer_t **consumer_p = (consumer_t **)luaL_checkudata(L, 1, consumer_label);
+    if (consumer_p == NULL || *consumer_p == NULL)
+        return 0;
+
+    if ((*consumer_p)->rd_consumer != NULL) {
+        int timeout_ms = lua_tonumber(L, 2);
```

Comment:
In `lua_consumer_list_groups` you are using an integer for the same timeout. Maybe it's better to use an integer here too.